manual/googlechrome-extensions/googlechrome-github-expandinizr/tools/install_x64.reg

```diff
@@ -0,0 +1,2 @@
+[HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432node\Google\Chrome\Extensions\cbehdjjcilgnejbpnjhobkiiggkedfib]
```

Comment:
Registry names are not cases sensitive. However, any application may treat them as it likes.